### ISSUE TITLE
Make the panel slide-in faster

### ISSUE THREAD

**Issue by damithc** [GitHubExtensions/Git-Labels#11]
The extension is for users who want to do things fast. The slide-in speed of the panel is a bit too slow for such users. I find myself already typing label names while in the panel is still sliding-in, and sometimes it result in unexpected behavior.

**Comment by jiajunGit** [comment]
@damithc I have added a fix for this issue in commit [4de1946](../commit/4de19463b14b9787949c83998f9c324f9d139ab8). Could you check to see if the input gains focus quick enough when the panel is sliding in? Thanks. 

Besides the panel sliding out due to the input not gaining focus fast enough when typing, are there any other unexpected behaviors you have experienced due to this issue?

**Comment by damithc** [comment]
~~If I type `L`, wait for panel to appear, and type `L` again: `p.Low` is selected, as expected.
If I type `L` and `L` without waiting, the panel aborts the slide-in halfway and slides out again.~~

**Comment by damithc** [comment]
Seems to be fixed. It seems I didn't update properly when I wrote the previous comment. 👍

**Comment by damithc** [comment]
The problem still persists if I type fast enough. e.g. If you type `L` and `A` fast enough, the labels panels slide in is aborted halfway and the `assignees` field in the page get activated instead.

**Comment by jiajunGit** [comment]
@damithc Could you check to see if this issue is resolved on your side? Thanks! (The fix should be in version 1.0.1)

**Comment by damithc** [comment]
Yes, seems so. First I have to wait till the panel icon appears, but after that even a very fast `L A` gets me the right label.  The problem persists if I type `L A` _before_ the panel icon appears, but that is unavoidable I guess.
